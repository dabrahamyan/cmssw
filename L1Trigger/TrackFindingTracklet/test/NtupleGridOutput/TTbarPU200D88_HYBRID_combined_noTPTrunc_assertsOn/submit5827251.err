************************************************************************************************************************
						 WARNING!
 The key phase2_realistic now points to 125X_mcRun4_realistic_v5.
 Usage of this key is discretionary and users are cautioned to use it at their own risk!
 This Global Tag contains tags suited for an Inner Tracker layout with rectangular (25x100um2) pixel cells (T15,T21).
 If this is not the geometry you are expecting to use, please consult:
 https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoCondPhase2.py to retrieve the right key.
************************************************************************************************************************
03-Sep-2023 13:32:46 CEST  Initiating request to open file root://cms-xrd-global.cern.ch///store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/81797c08-d712-4ee2-b013-7fcb5fcbbe24.root
03-Sep-2023 13:32:56 CEST  Successfully opened file root://cms-xrd-global.cern.ch///store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/81797c08-d712-4ee2-b013-7fcb5fcbbe24.root
%MSG-w XrdAdaptorLvl3:  AfterFile 03-Sep-2023 13:33:13 CEST pre-events
Deactivating heplnx229.pp.rl.ac.uk from active sources because its quality (861) is higher than 260 and 4 times larger than the other active server heplnx228.pp.rl.ac.uk (19) 
%MSG
L1TrackNtupleMaker::beginJob
Begin processing the 1st record. Run 1, Event 5202, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:31.624 CEST
Begin processing the 2nd record. Run 1, Event 5201, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:34.326 CEST
Begin processing the 3rd record. Run 1, Event 5203, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:36.738 CEST
Begin processing the 4th record. Run 1, Event 5204, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:40.676 CEST
Begin processing the 5th record. Run 1, Event 5206, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:43.401 CEST
Begin processing the 6th record. Run 1, Event 5205, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:46.694 CEST
Begin processing the 7th record. Run 1, Event 5207, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:49.334 CEST
Begin processing the 8th record. Run 1, Event 5208, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:52.561 CEST
Begin processing the 9th record. Run 1, Event 5209, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:55.205 CEST
Begin processing the 10th record. Run 1, Event 5211, LumiSection 53 on stream 0 at 03-Sep-2023 13:33:58.020 CEST
Begin processing the 11th record. Run 1, Event 5210, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:00.656 CEST
Begin processing the 12th record. Run 1, Event 5212, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:03.504 CEST
Begin processing the 13th record. Run 1, Event 5213, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:05.997 CEST
Begin processing the 14th record. Run 1, Event 5215, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:09.161 CEST
Begin processing the 15th record. Run 1, Event 5214, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:11.279 CEST
Begin processing the 16th record. Run 1, Event 5216, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:14.501 CEST
Begin processing the 17th record. Run 1, Event 5218, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:16.984 CEST
Begin processing the 18th record. Run 1, Event 5217, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:19.638 CEST
Begin processing the 19th record. Run 1, Event 5219, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:21.804 CEST
Begin processing the 20th record. Run 1, Event 5220, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:24.768 CEST
Begin processing the 21st record. Run 1, Event 5221, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:27.560 CEST
Begin processing the 22nd record. Run 1, Event 5223, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:30.502 CEST
Begin processing the 23rd record. Run 1, Event 5222, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:32.947 CEST
Begin processing the 24th record. Run 1, Event 5225, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:35.975 CEST
Begin processing the 25th record. Run 1, Event 5224, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:38.898 CEST
Begin processing the 26th record. Run 1, Event 5226, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:42.009 CEST
Begin processing the 27th record. Run 1, Event 5227, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:44.370 CEST
Begin processing the 28th record. Run 1, Event 5229, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:47.313 CEST
Begin processing the 29th record. Run 1, Event 5228, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:49.824 CEST
Begin processing the 30th record. Run 1, Event 5230, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:53.057 CEST
Begin processing the 31st record. Run 1, Event 5231, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:55.758 CEST
Begin processing the 32nd record. Run 1, Event 5233, LumiSection 53 on stream 0 at 03-Sep-2023 13:34:58.815 CEST
Begin processing the 33rd record. Run 1, Event 5232, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:01.595 CEST
Begin processing the 34th record. Run 1, Event 5235, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:04.760 CEST
Begin processing the 35th record. Run 1, Event 5234, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:07.277 CEST
Begin processing the 36th record. Run 1, Event 5236, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:10.496 CEST
Begin processing the 37th record. Run 1, Event 5237, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:12.516 CEST
Begin processing the 38th record. Run 1, Event 5238, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:15.339 CEST
Begin processing the 39th record. Run 1, Event 5239, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:18.222 CEST
Begin processing the 40th record. Run 1, Event 5240, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:21.767 CEST
Begin processing the 41st record. Run 1, Event 5241, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:24.443 CEST
Begin processing the 42nd record. Run 1, Event 5242, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:27.385 CEST
Begin processing the 43rd record. Run 1, Event 5243, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:29.444 CEST
Begin processing the 44th record. Run 1, Event 5244, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:32.679 CEST
Begin processing the 45th record. Run 1, Event 5245, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:35.293 CEST
Begin processing the 46th record. Run 1, Event 5246, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:38.182 CEST
Begin processing the 47th record. Run 1, Event 5247, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:40.484 CEST
Begin processing the 48th record. Run 1, Event 5248, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:43.271 CEST
Begin processing the 49th record. Run 1, Event 5250, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:46.196 CEST
Begin processing the 50th record. Run 1, Event 5249, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:49.101 CEST
Begin processing the 51st record. Run 1, Event 5252, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:51.984 CEST
Begin processing the 52nd record. Run 1, Event 5251, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:55.094 CEST
Begin processing the 53rd record. Run 1, Event 5254, LumiSection 53 on stream 0 at 03-Sep-2023 13:35:57.585 CEST
Begin processing the 54th record. Run 1, Event 5253, LumiSection 53 on stream 0 at 03-Sep-2023 13:36:00.528 CEST
Will break on number of tracklets in TP_L1L2K
cmsRun: /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/src/L1Trigger/TrackFindingTracklet/src/TrackletProcessor.cc:345: void trklet::TrackletProcessor::execute(unsigned int, double, double): Assertion `0' failed.


A fatal system signal has occurred: abort signal
The following is the call stack containing the origin of the signal.


Current Modules:

Module: L1FPGATrackProducer:l1tTTTracksFromTrackletEmulation (crashed)

A fatal system signal has occurred: abort signal
condor_exec.exe: line 7:   155 Aborted                 (core dumped) cmsRun L1TrackNtupleMaker_cfg_grid_HYBRID_trkJets.py inputFiles=root://cms-xrd-global.cern.ch///store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/81797c08-d712-4ee2-b013-7fcb5fcbbe24.root outputFile=/eos/user/d/dabraham/L1NtupleTrackExamples/TTbarPU200D88_HYBRID_combined_noTPTrunc_assertsOn_9.root maxEvents=10000
