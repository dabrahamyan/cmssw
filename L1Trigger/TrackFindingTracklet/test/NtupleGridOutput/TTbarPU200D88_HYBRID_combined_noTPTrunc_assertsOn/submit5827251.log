000 (5827251.000.000) 09/03 13:21:07 Job submitted from host: <137.138.44.55:9618?addrs=137.138.44.55-9618+[2001-1458-d00-1a--48]-9618&alias=bigbird17.cern.ch&noUDP&sock=schedd_7840_2a97>
...
022 (5827251.000.000) 09/03 13:22:35 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1_20@b7g12p2942.cern.ch <128.142.132.16:9618?addrs=128.142.132.16-9618+[2001-1458-301-5a--100-4]-9618&alias=b7g12p2942.cern.ch&noUDP&sock=startd_3892_2ab0>
...
024 (5827251.000.000) 09/03 13:22:35 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1_20@b7g12p2942.cern.ch, rescheduling job
...
022 (5827251.000.000) 09/03 13:32:25 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1_1@b7s11p0655.cern.ch <188.184.162.154:9618?addrs=188.184.162.154-9618+[2001-1458-301-4a--100-94]-9618&alias=b7s11p0655.cern.ch&noUDP&sock=startd_3262_0a06>
...
024 (5827251.000.000) 09/03 13:32:25 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1_1@b7s11p0655.cern.ch, rescheduling job
...
040 (5827251.000.000) 09/03 13:32:37 Started transferring input files
	Transferring to host: <128.142.132.166:9618?addrs=128.142.132.166-9618+[2001-1458-301-5a--100-9a]-9618&alias=b7g12p2799.cern.ch&noUDP&sock=slot1_6_8470_a192_147340>
...
040 (5827251.000.000) 09/03 13:32:37 Finished transferring input files
...
001 (5827251.000.000) 09/03 13:32:38 Job executing on host: <128.142.132.166:9618?addrs=128.142.132.166-9618+[2001-1458-301-5a--100-9a]-9618&alias=b7g12p2799.cern.ch&noUDP&sock=startd_7433_ff8b>
...
006 (5827251.000.000) 09/03 13:32:47 Image size of job updated: 185240
	181  -  MemoryUsage of job (MB)
	185240  -  ResidentSetSize of job (KB)
...
006 (5827251.000.000) 09/03 13:36:24 Image size of job updated: 2426608
	181  -  MemoryUsage of job (MB)
	185240  -  ResidentSetSize of job (KB)
...
040 (5827251.000.000) 09/03 13:36:24 Started transferring output files
...
040 (5827251.000.000) 09/03 13:36:24 Finished transferring output files
...
005 (5827251.000.000) 09/03 13:36:24 Job terminated.
	(1) Normal termination (return value 134)
		Usr 0 00:02:39, Sys 0 00:00:13  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:02:39, Sys 0 00:00:13  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	26375  -  Run Bytes Sent By Job
	700  -  Run Bytes Received By Job
	26375  -  Total Bytes Sent By Job
	700  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       41        1   1622803 
	   Memory (MB)          :      181     2000      2000 

	Job terminated of its own accord at 2023-09-03T11:36:24Z.
...
