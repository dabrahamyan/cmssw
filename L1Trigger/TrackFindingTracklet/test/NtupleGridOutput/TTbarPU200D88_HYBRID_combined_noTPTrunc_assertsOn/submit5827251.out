using geometry D88 (tilted)
Sun Sep  3 13:36:02 CEST 2023
Thread 7 (Thread 0x2ab4ab249700 (LWP 184) "cmsRun"):
#0  0x00002ab47771cb3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002ab47771cbcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002ab47771cc6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002ab47e1f9576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002ab47e1f9639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002ab477716ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002ab477a29b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2ab4ab048700 (LWP 183) "cmsRun"):
#0  0x00002ab47771cb3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002ab47771cbcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002ab47771cc6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002ab47e1f9576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002ab47e1f9639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002ab477716ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002ab477a29b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2ab4aae47700 (LWP 182) "cmsRun"):
#0  0x00002ab47771cb3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002ab47771cbcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002ab47771cc6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002ab47e1f9576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002ab47e1f9639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002ab477716ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002ab477a29b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2ab4aac46700 (LWP 181) "cmsRun"):
#0  0x00002ab47771de9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002ab47df3610f in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002ab47e1672d3 in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#3  0x00002ab47e167429 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002ab477716ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002ab477a29b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2ab4aaa45700 (LWP 180) "cmsRun"):
#0  0x00002ab477a2a0e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002ab47df305c2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002ab47df2cd4d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#3  0x00002ab47df35818 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#4  0x00002ab477716ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002ab477a29b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2ab49b16e700 (LWP 173) "cmsRun"):
#0  0x00002ab47771e1d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002ab47e856ad7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002ab47e8576ca in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002ab4772f2f90 in std::execute_native_thread_routine (__p=0x2ab48f4d9e30) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002ab477716ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002ab477a29b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2ab479aace40 (LWP 155) "cmsRun"):
#0  0x00002ab477a1eddd in poll () from /lib64/libc.so.6
#1  0x00002ab47e856dcf in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002ab47e85779c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002ab47e85a24b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002ab477961387 in raise () from /lib64/libc.so.6
#6  0x00002ab477962a78 in abort () from /lib64/libc.so.6
#7  0x00002ab47795a1a6 in __assert_fail_base () from /lib64/libc.so.6
#8  0x00002ab47795a252 in __assert_fail () from /lib64/libc.so.6
#9  0x00002ab4a865c331 in trklet::TrackletProcessor::execute(unsigned int, double, double) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#10 0x00002ab4a85d8b59 in trklet::Sector::executeTP() () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#11 0x00002ab4a8649263 in trklet::TrackletEventProcessor::event(trklet::SLHCEvent&, std::vector<std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >, std::allocator<std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > >&, std::vector<std::vector<trklet::StubStreamData, std::allocator<trklet::StubStreamData> >, std::allocator<std::vector<trklet::StubStreamData, std::allocator<trklet::StubStreamData> > > >&) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#12 0x00002ab4a8483a74 in L1FPGATrackProducer::produce(edm::Event&, edm::EventSetup const&) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#13 0x00002ab4753389ae in edm::one::EDProducerBase::doEvent(edm::EventTransitionInfo const&, edm::ActivityRegistry*, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#14 0x00002ab47532223f in edm::WorkerT<edm::one::EDProducerBase>::implDo(edm::EventTransitionInfo const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#15 0x00002ab475272b25 in decltype ({parm#1}()) edm::convertException::wrap<edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}>(edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#16 0x00002ab475272e13 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(std::__exception_ptr::exception_ptr, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#17 0x00002ab475275b5a in void edm::SerialTaskQueueChain::actionToRun<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&>(edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#18 0x00002ab475275dd1 in edm::SerialTaskQueue::QueuedTask<edm::SerialTaskQueueChain::push<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&>(tbb::detail::d1::task_group&, edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&)::{lambda()#1}>::execute() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#19 0x00002ab47500d175 in tbb::detail::d1::function_task<edm::SerialTaskQueue::spawn(edm::SerialTaskQueue::TaskBase&)::{lambda()#1}>::execute(tbb::detail::d1::execution_data&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreConcurrency.so
#20 0x00002ab476b08bec in tbb::detail::r1::task_dispatcher::local_wait_for_all<false, tbb::detail::r1::external_waiter> (waiter=..., t=0x2ab47b23aa00, this=0x2ab47b267700) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:322
#21 tbb::detail::r1::task_dispatcher::local_wait_for_all<tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x2ab47b267700) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:463
#22 tbb::detail::r1::task_dispatcher::execute_and_wait (t=<optimized out>, wait_ctx=..., w_ctx=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.cpp:168
#23 0x00002ab4751ed22d in edm::FinalWaitingTask::wait() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#24 0x00002ab4751d5310 in edm::EventProcessor::processRuns() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#25 0x00002ab4751e1da1 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#26 0x000000000040a266 in tbb::detail::d1::task_arena_function<main::{lambda()#1}::operator()() const::{lambda()#1}, void>::operator()() const ()
#27 0x00002ab476af70eb in tbb::detail::r1::task_arena_impl::execute (ta=..., d=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/arena.cpp:698
#28 0x000000000040b094 in main::{lambda()#1}::operator()() const ()
#29 0x000000000040971c in main ()
